packages/forms/src/useFormikEnhanced.ts

```diff
@@ -25,18 +25,18 @@ export interface AppFormikConfig<TValues extends FormikValues, TResponse>
   onSubmitFailure?: (error: Error, values: TValues) => void;
 }
 
-export type AppFormikStatus<TResponse> =
+export type FormikEnhancedStatus<TResponse> =
   | { type: 'initial'; payload?: undefined }
   | { type: 'submitted'; payload: TResponse }
   | { type: 'rejected'; payload: Error };
 
-export interface AppFormik<TValues extends FormikValues, TResponse>
+export interface FormikEnhanced<TValues extends FormikValues, TResponse>
```

Comment:
We're not enhancing formik in any way, we just add typing for `status` and `setStatus`. It should `FormikContextTypeWithTypedStatus`, and with some todo that says that this should be removed once https://github.com/jaredpalmer/formik/pull/2323 will be merged and released.

Reply:
Maybe `FormikContextTypeEnhanced` ?

Reply:
@doniyor2109 I like it!